### PR TITLE
refactor: use API from env variable instead of hardcoded URL

### PR DESCRIPTION
**Pull request by blenzi**
- Replace https://api.pyronear.org by $API_URL
- style: fix headers, black and mypy corrections (add `__init__.py` files)
- upgrade certifi

### CODE REVIEW

**Comment by blenzi** [comment]
The secrets are most likely outdated and that is probably why the tests are failing. See for instance: https://github.com/pyronear/pyro-platform/actions/runs/4682835248/jobs/8297190366?pr=91#step:4:40

File: app/components/__init__.py
Comment (frgfm) [inline app/components/__init__.py:1]: Far from important but: is this necessary? The platform was running beforehand

**Review by frgfm — COMMENTED** [review]
Looks good, thanks!
Before merging, I suggest two things:
- pinning the black version in the CI to the previous version (happy to open a PR) & revert the removed spaces in this PR. Otherwise every time black changes, we'll have the same problem while the goal is to have consistent formmating
- very optional: for the copyright, it's normal that everything wasn't starting in 2020 as the file was created afterwards

File: app/components/__init__.py
Comment (blenzi) [reply app/components/__init__.py:1]: According to mypy, yes: https://github.com/pyronear/pyro-platform/pull/91/commits/7439ff31563dfb0d8836fee06f60da1b7238b12e

File: requirements.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 blinker==1.4
 brotli==1.0.9; python_version >= "2.7" and python_full_version < "3.0.0" or python_full_version >= "3.3.0"
 cachelib==0.9.0; python_version >= "3.7"
-certifi==2022.6.15; python_version >= "3.7" and python_version < "4"
+certifi===2022.12.7; python_version >= "3.7" and python_version < "4"
```
[the remark this answers is not in the document]
Comment (blenzi) [reply requirements.txt:4]: I was getting a problem with SSL. But maybe it was fixed in another way or maybe this was the solution, I am not sure anymore. If you want I can test with the previous version but I would say it is harmless

**Comment by blenzi** [comment]
Thanks @frgfm ! 
- Happy to go back to the previous version of black but not sure when it was changed and which version was used. So if you can do the PR, even better
- For the copyright, this is what the headers check impose

File: requirements.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (frgfm) [reply requirements.txt:4]: Ok thanks! One last thing: is this normal that there is a triple equal sign? (instead of a double?)

**Comment by frgfm** [comment]
> * Happy to go back to the previous version of black but not sure when it was changed and which version was used. So if you can do the PR, even better

Yup I'll take care of this tomorrow :+1: 

> * For the copyright, this is what the headers check impose

My bad, the message doesn't give a dynamic suggestion per file, I'll do this at the same time. Then we'll be able to merge main into this PR branch and we'll be good to go :ok_hand:

**Review by Akilditu — APPROVED** [review]
Hi @blenzi, 

Thanks for the PR ! 

Indeed needed as URLs were old API ones, small changes still needed in another PR to make everything aligned and ready for OVH migration.

**Comment by blenzi** [comment]
Thanks @Akilditu . @frgfm, we need to move forward asap to have the platform running. So I will merge the PR and we can follow the discussion about the code quality checks on #92 .

**Pull request merged**